README.md

```diff
@@ -50,14 +65,19 @@ Staged resource drafts are automatically tested:
 
 Tests can also be triggered (via github api or manually) by dispatching the [test workflow][test-wf][^2].
 
-Once the tests are completed, a reviewer needs to take a look.
-The uploader gets a notification via email.
+Once the tests are completed, the uploader gets a notification via email that their draft is awaiting review; now a reviewer needs to take a look.
+
 An overview of all pending _resource drafts_ can be found at https://bioimageio-uploader.netlify.app/#/status.
 A _draft_ is identified by its concept id (`id` from the `rdf.yaml`).
 
 #### Review
 
 Reviewers should check the models for technical correctness (aided by CI, see [Testing section](#testing)) and contents/metadata of the resource.
+
+To this end it can be helpful to check the logs displayed at https://bioimageio-uploader.netlify.app/#/status/<concept id>.
```

Comment:
for me the render preview would not add <concept id> but who knows... web

Reply:
fixed (I guess it was the space.. I also added back ticks to mark it as invalid url that needs editing